geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -155,11 +167,12 @@ public ClusterManagementResult update(CacheElement config) {
 
   @Override
   public ClusterManagementResult list(CacheElement filter) {
-    ConfigurationManager manager = managers.get(filter.getClass());
+    ConfigurationManager<CacheElement, RuntimeCacheElement> manager =
+        managers.get(filter.getClass());
```

Comment:
this line gives me an "unchecked assignment" warning.  can we add an explicit cast or some annotation to suppress this warning (or see if there's any correct way to avoid it, perhaps similar to TYPESAFE HETEROGENEOUS CONTAINERS (item 33) in Effective Java 3rd Edition?)